### PR TITLE
update changelog, add breaking changes, bump version

### PR DESCRIPTION
**Pull request by gimre-xymcity**

### CODE REVIEW

File: BREAKING-CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    'type': 'transfer',
+    'signer_public_key': 'TEST',
+    'fee': 1000000,
+    'deadline': 41998024783,
+    'recipient_address': 'TD4PJKW5JP3CNHA47VDFIM25RCWTWRGT45HMPSA',
+    'mosaics': [(0x2CF403E85507F39E, 1000000)]
+```
+**NEW**:
+```py
+	'type': 'transfer',
```
Comment (Jaguar0625) [inline BREAKING-CHANGES.md:26]: tab => space

File: BREAKING-CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+3. Signature should always be attached via transaction factory, sample:
+```py
+signature = facade.sign_transaction(key_pair, transaction)
+facade.transaction_factory.attach_signature(transaction, signature)
+```
+
+### Generated code 
+
+1. Catbuffer python package has been completely deprecated, low-level structures are available under SDK's `symbolchain.sc` module
```
Comment (Jaguar0625) [inline BREAKING-CHANGES.md:44]: provide full package name?

File: BREAKING-CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+2. Structure of a module has been flattened, Dto suffix has been removed:
```
Comment (Jaguar0625) [inline BREAKING-CHANGES.md:46]: `Structure of a module has been flattened` hm?

File: setup.py [hunk 4]
```diff
@@ -9,7 +9,6 @@
     VERSION = version_file.read().strip()
 
 REQUIRES = [
```
Comment (Jaguar0625) [inline setup.py:11]: update requirements.txt too?

File: BREAKING-CHANGES.md (hunk 2 above)
Comment (gimre-xymcity) [reply BREAKING-CHANGES.md:44]: hm? that's a full package name, what do you mean?

File: setup.py (hunk 4 above)
Comment (gimre-xymcity) [reply setup.py:11]: it somehow was updated earlier...

File: BREAKING-CHANGES.md (hunk 2 above)
Comment (Jaguar0625) [reply BREAKING-CHANGES.md:44]: pip name - `catbuffer`

File: BREAKING-CHANGES.md (hunk 3 above)
Comment (Jaguar0625) [reply BREAKING-CHANGES.md:46]: you're referring to just catbuffer replacement (i.e. generated code) or _everything_ ?

Comment (gimre-xymcity) [reply BREAKING-CHANGES.md:46]: this is section `### Generated code ` 😁

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**